sprint01/task06/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Oscorp</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <a href="" class="header__logo">OSCORP</a>
+            <nav class="header__menu">
+                <ul class="header__list">
+                    <li>
+                        <a href="" class="header__link">About</a>
+                    </li>
+                    <li>
+                        <a href="" class="header__link">Service</a>
+                    </li>
+                    <li>
+                        <a href="" class="header__link">Contact</a>
+                    </li>
+                    <li>
+                        <input class="header__link" placeholder="Search" type="text">
+                    </li>
+                    <li>
+                        <a href="" class="header__link">Phone:+777777777</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <nav>
+            <ul class="breadcrumb">
+               <li><a href="">Home</a></li>
+               <li><a href="file:///home/igor/ucode/sprint01/task06/index3.html">Catalog</a></li>
+               <li><a href="file:///home/igor/ucode/sprint01/task06/card.html">Product</a></li>
```

Comment:
The way you put a path here (`file:///home/igor/ucode/sprint01/task06/card.html`) will work only on your system. On mine it does not work. Instead, use relative path like this: `./card.html`.